tests/function_libs/torch_lib/extra_opinfo.py

```diff
@@ -2308,7 +2308,7 @@ def __init__(self):
     opinfo_core.OpInfo(
         "ops.aten._local_scalar_dense",
         aten_name="_local_scalar_dense",
-        dtypes=common_dtype.all_types(),
+        dtypes=common_dtype.all_types() + (torch.bool,),
```

Comment:
```suggestion
        dtypes=common_dtype.all_types_and (torch.bool),
```

maybe this?

Reply:
https://github.com/xadupre/onnxscript/blob/2aefdc9c009934e4f9393802fe68a9265c2d2e98/tests/function_libs/torch_lib/ops_test.py#L11

`pytest onnxscript/tests/function_libs/torch_lib/ops_test.py -k _local_scalar_dense`